kong/db/declarative/init.lua

```diff
@@ -282,6 +282,56 @@ function declarative.export_from_db(fd)
 end
 
 
+function declarative.export_config()
```

Comment:
Yeah, let's address this in rc2 as it is not user visible.

Reply:
Yep, I was going to make the same comment. Not just different names, but this needs to be refactored to avoid the actual code duplication. But agreed, it's minor and not a blocker.